test/integration/telemetry/test/index.test.js

```diff
@@ -619,12 +619,14 @@ describe('Telemetry CLI', () => {
     const optimizeFonts = regex.exec(stderr).pop()
     expect(optimizeFonts).toContain(`"featureName": "optimizeFonts"`)
     expect(optimizeFonts).toContain(`"invocationCount": 1`)
-    const swcLoader = regex.exec(stderr).pop()
-    expect(swcLoader).toContain(`"featureName": "swcLoader"`)
-    expect(swcLoader).toContain(`"invocationCount": 1`)
-    const swcMinify = regex.exec(stderr).pop()
-    expect(swcMinify).toContain(`"featureName": "swcMinify"`)
-    expect(swcMinify).toContain(`"invocationCount": 0`)
+    regex.exec(stderr).pop() // swcLoader
+    regex.exec(stderr).pop() // swcMinify
+    regex.exec(stderr).pop() // swcRelay
+    regex.exec(stderr).pop() // swcStyledComponents
+    regex.exec(stderr).pop() // swcExperimentalDecorators
+    regex.exec(stderr).pop() // swcReactRemoveProperties
+    regex.exec(stderr).pop() // swcRemoveConsole
+    regex.exec(stderr).pop() // swcImportSource
```

Comment:
Should we be checking for `toContain()` on each of these?

Reply:
Oh maybe we should just combine this test and the new test you added below?

Reply:
No, the next test is testing that they are counted correctly

Reply:
Well there's more than 2 tests that do it this way, so if we combine these two we may as well combine them all and it would be one giant test. Maybe we could try filter instead pop.